symbols/overlay00.yml

```diff
@@ -15,5 +15,12 @@ overlay0:
     Likely contains supporting data and code related to the top menu.
     
     This is loaded together with overlay 1 while in the top menu. Since it's in overlay group 2 (together with overlay 10, which is another "data" overlay), this overlay probably plays a similar role. It mentions several files from the BACK folder that are known backgrounds for the top menu.
-  functions: []
+  functions:
+    - name: SelectRandomBackground
+        address:
```

Comment:
These are indented too far. `address` and `description` should be at the same level as `name`